python/lib/cloudutils/syscfg.py

```diff
@@ -222,6 +224,18 @@ def __init__(self, glbEnv):
                          nfsConfig(self),
                          cloudAgentConfig(self)]
 
+#it covers RHEL9
+class sysConfigRedhat9(sysConfigAgentRedhat8Base):
+    def __init__(self, glbEnv):
+        super(sysConfigRedhat9, self).__init__(glbEnv)
+        self.services = [hostConfig(self),
+                         securityPolicyConfigRedhat(self),
+                         networkConfigRedhat(self),
+                         libvirtConfigRedhat(self),
+                         firewallConfigAgent(self),
+                         nfsConfig(self),
+                         cloudAgentConfig(self)]
+
```

Comment:
this is exactly the same as sysConfigRedhat7 and sysConfigRedhat8 the extra code adds nothing here.

Reply:
I don't know - that's why I said I followed the pattern. With python there's a lot of runtime/meta programming that happens.

Reply:
Done